x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_tables.tsx

```diff
@@ -126,13 +127,14 @@ export const RulesTables = React.memo<RulesTableProps>(
       onFinish: hideDeleteConfirmation,
     });
 
-    const [isBulkEditConfirmationVisible, showBulkEditConfirmation, hideBulkEditConfirmation] =
-      useBoolState();
-
-    const [confirmBulkEdit, handleBulkEditConfirm, handleBulkEditCancel] = useAsyncConfirmation({
-      onInit: showBulkEditConfirmation,
-      onFinish: hideBulkEditConfirmation,
-    });
+    const {
+      bulkActionsDryRunResult,
+      bulkAction,
+      isBulkActionConfirmationVisible,
+      showBulkActionConfirmation,
+      cancelBulkActionConfirmation,
+      approveBulkActionConfirmation,
+    } = useBulkActionsConfirmation();
```

Comment:
This component now exceeded the maximum complexity and eslint warning makes it hardly readable in the IDE:

<img width="1208" alt="Screenshot 2022-07-22 at 18 37 14" src="https://user-images.githubusercontent.com/7359339/180485065-95479dab-a38c-4ce8-9e3e-943699898972.png">

I'd try to first replace destructuring the hooks' results with a dot notation style of code. Hopefully, that would fix it.

Reply:
Looks it's the consequence of removing disabled complexity eslint rule in https://github.com/elastic/kibana/pull/134664/files#diff-a68eda48917a2670f954b236dfbdc0ad53a54bc045b25f72491c81871f0742fdL8-L9

Didn't last long.. So, adding it back here.

Reply:
Well, would be extra nice to simplify the implementation instead of disabling this eslint rule, but I'm ok with doing it in a separate PR. We have several places where eslint tells us that it's become a mess, but we don't fix it and instead disable it 🙈